src/Controller/Admin/BlogController.php

```diff
@@ -130,11 +131,10 @@ public function show(Post $post): Response
      * Displays a form to edit an existing Post entity.
      *
      * @Route("/{id}/edit", requirements={"id": "\d+"}, methods={"GET", "POST"}, name="admin_post_edit")
+     * @IsGranted("edit", subject="post", message="Posts can only be edited by their authors.")
      */
     public function edit(Request $request, Post $post): Response
     {
-        $this->denyAccessUnlessGranted('edit', $post, 'Posts can only be edited by their authors.');
```

Comment:
It would be good to keep this as another way of doing the same but without the annotation one.

Reply:
I let an example in the show action ([see the code](https://github.com/DevMDamien/demo/blob/137b484b25e788d43668ce352da420993d62529c/src/Controller/Admin/BlogController.php#L121-L123))